### ISSUE TITLE
imu data no longer streaming in SteamVR 1.16.8

### ISSUE THREAD

**Issue by TobiSchluter** [ValveSoftware/openvr#1530]
I had a previous bug report just a few days ago where I pointed out a bug in the IOBuffer / ImuSample_t documentation:
https://github.com/ValveSoftware/openvr/issues/1528

Now I updated to SteamVR 1.16.8 -- and the IMUs are gone. The call to `vr::VRIOBuffer()->Open` invariably returns IOBuffer_PathDoesNotExist. Did the name change? Is this an intended change? I'm trying to follow the example from this page, which used to work up until at least SteamVR 1.13.10 https://github.com/ValveSoftware/openvr/wiki/IVRIOBuffer

(I had a quick look around OpenXR since it was introduced in 1.16.8 but I found nothing that leads me to believe that I could access the IMU from there.)

**Comment by TobiSchluter** [comment]
It looks like the buffer has reappeared in 1.17 under a different name :open_mouth:

**Comment by JoeLudwig** [comment]
This interface is problematic for a bunch of reasons. Can you tell me what you're using it for? Maybe we can find a better way to accomplish those goals.

**Comment by JoeLudwig** [comment]
This data stream is back at the old path as of 1.17.5: https://steamcommunity.com/games/250820/announcements/detail/3102392357027214774

**Comment by TobiSchluter** [comment]
Hi,

thank you for fixing this so quickly. But let me answer your question nonetheless.

We are using it for our product LPVR https://lp-research.com/virtual-and-augmented-reality-tracking-solutions/ which combines SteamVR headsets on the one hand with large-scale tracking solutions such as AR-Tracking for applications such as location-based VR or design presentations at car manufacturers, and on the other with moving platforms such as simulators, cars or aircraft.

Before this interface became available, we used to sell one of our IMUs together with the set, but since the IMU data became available our system became a lot simpler. Now, for 1.16 specifically I had to listen to the IMU HID device which has a nice advantage in that the timing is tighter because of the interrupt-driven nature, but which has some disadvantages as well (I couldn’t find how to do the complete clock-sync that the IOBuffer offers at least on Windows, I still need the calibration parameters from SteamVR, it’s not a documented interface like this one, so it could stop working any minute j/k).

Anyway, if you are interested, I think we should set up a call. Obviously we are very much interested in you being aware of our requirements.
